temporalio/lib/temporalio/internal/worker/workflow_worker.rb

```diff
@@ -13,7 +13,7 @@ module Internal
     module Worker
       # Worker for handling workflow activations. Most activation work is delegated to the workflow executor.
       class WorkflowWorker
-        def self.workflow_definitions(workflows)
+        def self.workflow_definitions(workflows, should_enforce_versioning_behavior: false)
```

Comment:
```suggestion
        def self.workflow_definitions(workflows, should_enforce_versioning_behavior:)
```

I've found it safer for internal methods to not have default parameters and rather force all internal callers to clearly specify the parameters